core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -155,6 +157,10 @@ private TachyonFS(InetSocketAddress masterAddress, boolean zookeeperMode) throws
     mMasterClient =
         mCloser.register(new MasterClient(mMasterAddress, mZookeeperMode, mExecutorService));
     mWorkerClient = mCloser.register(new WorkerClient(mMasterClient, mExecutorService));
+
+    String scheme = mZookeeperMode ? Constants.SCHEME_FT : Constants.SCHEME;
+    String authority = mMasterAddress.getHostName() + ":" + mMasterAddress.getPort();
```

Comment:
if mMasterAddress.toString() returns "/127.0.0.1:19998", after `mMasterAddress.getHostName()` , `mMasterAddress.toString()` will return "localhost/127.0.0.1:19998", this will affect `toStringTest`, so the unit test needs to be modified.